### DISCUSSION TITLE
How to filter notifications of boosted and favourited posts

### DISCUSSION THREAD

**Discussion by MSSProvenance** [mastodon/mastodon#22972] [Q&A] [answered]
I don't want my Notifications feed filled-up with notifications that my posts have been "boosted" or "favourited". 
How do I filter these out?
Thanks in advance.

**Comment by trwnh** [comment, marked as answer]
The Notifications column should have a settings button in the header which lets you customize which notifications to receive and whether they should make sound, etc.

**Reply by MSSProvenance** [reply]
Yes! Thank you. So excluding unwanted mentions is a "setting" not a "filter"!
